show-portal-user-info/src/main/java/com/esri/arcgismaps/sample/showportaluserinfo/MainActivity.kt

```diff
@@ -35,6 +37,11 @@ class MainActivity : ComponentActivity() {
         // required to access basemaps and other location services
         ArcGISEnvironment.apiKey = ApiKey.create(BuildConfig.API_KEY)
```

Comment:
Should we be setting an API key here, since the sample uses a named user?

\cc @shubham7109